src/main/java/bio/terra/cli/command/config/set/Image.java

```diff
@@ -14,10 +14,10 @@
   ImageIdArgGroup argGroup;
 
   static class ImageIdArgGroup {
-    @CommandLine.Option(names = "--image", description = "image id or tag")
+    @CommandLine.Option(names = "--image", description = "Docker image id or tag.")
     private String imageId;
 
-    @CommandLine.Option(names = "--default", description = "use the default image id or tag")
+    @CommandLine.Option(names = "--default", description = "Use the default image id or tag.")
```

Comment:
indicate how these two options work together, e.g. default overrides `--image`.

Reply:
You can only specify one at a time, so there's no overriding. picocli indicates this relationship for us in the auto-generated usage help.
```
> terra config set image
Error: Missing required argument (specify one of these): (--image=<imageId> | --default)
Usage: terra config set image (--image=<imageId> | --default)
```